spec/views/external_users/claim_types/selection_spec.rb

```diff
@@ -1,6 +1,6 @@
 require 'rails_helper'
 
-describe 'external_users/claim_types/selection.html.haml', type: :view do
+RSpec.describe 'external_users/claim_types/selection.html.haml', type: :view do
```

Comment:
I'm unsure of the value this kind of specs brings us 😰 . Happy to discuss it with the team though.

Reply:
I've never seen the value in view specs either...